src/components/header/modals/ModalAccountUnification.vue

```diff
@@ -40,6 +72,25 @@ export default defineComponent({
   setup(props, { emit }) {
     const isSelected = ref<boolean>(false);
     const isClosing = ref<boolean>(false);
+    const currentStep = ref<number>(0);
+
+    const modalTitle = computed(() => {
```

Comment:
const modalTitle = computed((): string => {

Reply:
Fixed it